lib/seed/user_seeder.rb

```diff
@@ -11,15 +13,36 @@ def initialize(config:)
       @logger = Rails.logger.child(class: self.class.name)
       @number_of_users_per_facility = config.max_number_of_users_per_facility
       @organization = Seed.seed_org
-      puts "Starting #{self.class} with #{config.type} configuration"
+      announce "Starting #{self.class} with #{config.type} configuration"
     end
 
     attr_reader :config
     attr_reader :logger
     attr_reader :number_of_users_per_facility
     attr_reader :organization
+    delegate :stdout, to: :config
 
     def call
+      create_dashboard_admins
+      create_mobile_users
+    end
+
+    private
+
+    def create_dashboard_admins
+      unless EmailAuthentication.exists?(email: "power_user@simple.org")
+        FactoryBot.create(:admin, :power_user, full_name: "Power User", email: "power_user@simple.org", password: config.admin_password)
+      end
```

Comment:
What happens if a `power_user@simple.org` exists, but they are not a power user? I know it's unlikely, but more robustness here means we won't have to babysit this process as often.

Reply:
We can definitely make this more idempotent, but I'd like to do that in future PRs around this.  Creating some stock users with minimal idempotency is an improvement over our existing setup where don't have any stock admin users. 😁 

Right now Sandbox seed data is empty and I want to fast track this so we can get it fixed.